modules/security-context-constraints-psa-alert-eval.adoc

```diff
@@ -26,17 +26,15 @@ $ oc adm must-gather -- /usr/bin/gather_audit_logs
 +
 [source,terminal]
 ----
-$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/quay*/audit_logs/kube-apiserver/*log.gz \
-  | jq -r 'select((.annotations["pod-security.kubernetes.io/audit-violations"] != null) and (.objectRef.resource=="pods")) | .objectRef.namespace + " " + .objectRef.name + " " + .objectRef.resource' \
+$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/<registry_id>/audit_logs/kube-apiserver/*log.gz \
+  | jq -r 'select((.annotations["pod-security.kubernetes.io/audit-violations"] != null) and (.objectRef.resource=="pods")) | .objectRef.namespace + " " + .objectRef.name' \
   | sort | uniq -c
 ----
 +
-Replace `must-gather.local.<archive_id>` with the actual directory name.
+Replace `<archive_id>` and `<registry_id>` with the actual path names.
```

Comment:
Minor nit: from my test example, `quay-io-openshift-release-dev-ocp-v4-0-art-dev-sha256-dda3...` looks like image, not registry. And precisely, it is not image id, is image digest `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dda3`'s reformat on several special characters.

Reply:
Thanks @xingxingxia, I changed <registry_id> to <image_digest_id>